src/plots/plotGroup.ts

```diff
@@ -0,0 +1,33 @@
+module Plottable.Plots {
+  export class PlotGroup extends Components.Group {
+    public entityNearest(point: Point): PlotEntity {
+      let closestPlotEntity: PlotEntity;
+      let minDistSquared = Infinity;
+      this.components().forEach((plot: Plot) => {
+        let candidatePlotEntity = plot.entityNearest(point);
+        if (candidatePlotEntity == null) {
+            return;
+        }
+
+        let position = candidatePlotEntity.position;
+        let distSquared = Math.pow(position.x - point.x, 2) + Math.pow(position.y - point.y, 2);
```

Comment:
This position is the centroid of the entity or is it the top-left point?

In either case, we should be considering the full bounds of the entity since the nearest entity might not be the entity with the nearest centroid.